jetty9-base/src/main/jetty-base/modules/gcp-gcloud.mod

```diff
@@ -0,0 +1,9 @@
+[provides]
+gcloud
+
+[lib]
+lib/gcloud/*.jar
+
+[ini]
+# Hide the gcloud libraries from deployed webapps
+jetty.webapp.addServerClasses+=,${jetty.base.uri}/lib/gcloud/
```

Comment:
We extended jetty's normal class hiding mechanism that is based on package names to also handle locations - for modules with many jar dependencies it is too error-prone/tedious to enumerate all of the packages.